### DISCUSSION TITLE
MacBook ProM1 macOS 12.4 dimmed the A1316 and doesn't set the brightness back

### DISCUSSION THREAD

**Discussion by battle-slug** [MonitorControl/MonitorControl#1115] [General]
Hello, 
I have MacBook ProM1 macOS 12.4 and Apple LED Cinema A1316.
I installed MonitorControl, because my Apple LED Cinema A1316 was quite dim, but I still could work with that brightness though it was obviously 50%. I had USB plugged in. However, the brightness control was showing that it's 100% brightness, and when I touched the control the brightness became very dim. So the picture on it is barely visible now though the MonitorControl slider shows it's 100%...
I tried rebooting, quitting the Monitor control, plugging in the display to a different machine, but nothing helps, the display is still super dim...

Can you help with this please?

**Comment by the0neyouseek** [comment]
Please see this discussion https://github.com/MonitorControl/MonitorControl/discussions/739

**Reply by battle-slug** [reply]
Thank you so much! That worked!
